airflow_client/docs/PoolApi.md

```diff
@@ -405,10 +401,6 @@ with client.ApiClient(configuration) as api_client:
     pool = Pool(
         name="name_example",
         slots=1,
-        occupied_slots=1,
-        used_slots=1,
-        queued_slots=1,
-        open_slots=1,
```

Comment:
Why is this all being removed?

Reply:
@kaxil I'm not sure, as this is auto-generated code.